polyfills/Number/isInteger/config.json

```diff
@@ -0,0 +1,15 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "*",
```

Comment:
The browsers list isn't quite complete, [we maintain a list of supported ones here](https://polyfill.io/v2/docs/contributing/authoring-polyfills#browsers). The [Kangax compatibility tables](kangax.github.io/compat-table/es6) are a good place to get this information.